test.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
```

Comment:
this file needs to live in a directory called tests and should be named the same as the location of the code that you are testing eg: functions.php